docs/pbc/all/data-exchange/202311.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md

```diff
@@ -465,6 +465,93 @@ to ensure accurate and consistent data manipulation during `PUT` operations.
 
 {% endinfo_block %}
 
+### Sending `POST`, `PATCH` and `PUT` requests with relationships
+
+To create or update an entity along with its related entities, you need to include the relationships directly in 
+the request payload. The payload should be structured to reflect the hierarchy and connections between the main entity 
+and its child entities.
+
+{% info_block infoBox %}
+
+Currently, our system does not support `many-to-many` relationships for POST, PATCH and PUT requests. 
+Only `one-to-one` and `one-to-many` relationships are allowed. This means each child entity can be associated 
+with only one parent entity.
+
+{% endinfo_block %}
+
+The payload for these requests follows a nested structure where the main entity and its related entities are included within a data array. 
+Each object in the data array represents an instance of the main entity, and each related entity is nested within it.
+
+To be processed correctly related entities must be defined with existing relation names, and they should be
+organized to align with their hierarchical relationships in the database. 
```

Comment:
It is still not clear what it means.
>  and they should be
organized to align with their hierarchical relationships in the database

Hierarchical relationships of what? Database tables? 

Please add more details.